framework/tlvf/tlvf.py

```diff
@@ -1556,16 +1555,16 @@ def addClassSizeMethod(self, insert_name, insert_marker, name):
     def closeObject(self, obj_meta):
         if obj_meta.type == MetaData.TYPE_CLASS: # add class constractor
             # constractor 1
-            self.insertLineH(obj_meta.name, self.CODE_CLASS_CONSTRACTOR, "%s(uint8_t* buff, size_t buff_len, bool parse = false, bool swap_needed = false);" % (obj_meta.name))
-            self.insertLineCpp(obj_meta.name, self.CODE_CLASS_CONSTRACTOR, ("%s::%s(uint8_t* buff, size_t buff_len, bool parse, bool swap_needed) :" % (obj_meta.name, obj_meta.name)))
-            self.insertLineCpp(obj_meta.name, self.CODE_CLASS_CONSTRACTOR, ("%sBaseClass(buff, buff_len, parse, swap_needed) {" % self.getIndentation(1) ))
+            self.insertLineH(obj_meta.name, self.CODE_CLASS_CONSTRACTOR, "%s(uint8_t* buff, size_t buff_len, bool parse = false);" % (obj_meta.name))
```

Comment:
framework/tlvf/README.md still refers to the swap_needed argument.

There's also one in son_slave_thread.cpp comment and in beerocks_message.cpp.

Reply:
Will be fixed as part of #601